src/components/studies/InstructionArea.tsx

```diff
@@ -194,15 +188,40 @@ const InstructionArea: React.FC<Props> = ({
             </a>
             on your machine.
           </p>
-          <div>
-            Click below to download <code>auth_key.txt</code> which you will need on your machine to authenticate with
-            the sfkit command-line interface.
-            <div className="text-center mt-2">
-              <button className="btn btn-primary btn-sm" onClick={handleDownloadAuthKey}>
-                Download Auth Key
-              </button>
-            </div>
-          </div>
+          <p>
+            { onTerra ? (
+              <>
+                If you are running the study on a machine <b><i>outside of Terra</i></b>, you
+              </>
+            ) : (
+              <>
+                On your machine, you
+              </>
+            )} will also need to download <code>
+            { onTerra ? "service_account_key.json" : "auth_key.txt" }
+            </code> { onTerra && "and run the following command " }
+            to authenticate the <i>sfkit</i> command-line interface:
+          </p>
+          { onTerra && (
+            <p className="p-2 rounded" style={{ backgroundColor: "#f0f0f0" }}>
+              <code>
+                export GOOGLE_APPLICATION_CREDENTIALS=/path/to/service_account_key.json
```

Comment:
This needs to come with a warning that these need to be protected as a secret. As the key is long lived and probably shouldn't be used at all if you can help it.